kubernetes/config/common.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+data:
+  common.libsonnet: |
+    {
+      blobstore: {
+        contentAddressableStorage: {
+          sharding: {
+            hashInitialization: 11946695773637837490,
+            shards: [
+              {
+                backend: { grpc: { address: 'storage-0.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+              {
+                backend: { grpc: { address: 'storage-1.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+            ],
+          },
+        },
+        actionCache: {
+          sharding: {
+            hashInitialization: 14897363947481274433,
+            shards: [
+              {
+                backend: { grpc: { address: 'storage-0.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+              {
+                backend: { grpc: { address: 'storage-1.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+            ],
+          },
+        },
+      },
+      browserUrl: 'http://localhost:7980',
```

Comment:
It might be wiser to pick another placeholder address here. `http://address-of-bb-browser-ingress/`? Maybe better to leave it commented out? That way users are required to set it explicitly based on their setup.